src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/index.jsx

```diff
@@ -208,6 +223,57 @@ const CustomDescription = ({ amount = 0, classes, data, recipient }: CustomDescP
   )
 }
 
+const BountyPerson = ({ label, value }: BountyPersonField) => {
+  const { addr, amount } = value
+  const recipientName = getNameFromAddressBook(addr)
+  const roundValue = bounty.formatValue(amount.value)
+  return (
+    <div>
+      <div style={{ fontWeight: 'bold' }}>{label}:</div>
+      <div>
+        Value: {roundValue} {amount.unit}
+      </div>
+      <div>
+        {recipientName ? (
+          <div style={{ display: 'flex' }}>
+            Address: {recipientName}
+            <div style={{ display: 'flex', marginLeft: '0.5rem' }}>
+              <CopyBtn content={addr} />
+              <EtherscanBtn type="address" value={addr} />
+            </div>
+          </div>
+        ) : (
+          <EtherscanLink knownAddress={false} type="address" value={addr} />
+        )}
+      </div>
+    </div>
+  )
+}
+
+const BountyTxDescription = ({ classes, data }: CustomDescProps) => {
+  return (
+    <>
+      <Block className={classes.txData} data-testid={TRANSACTIONS_DESC_CUSTOM_DATA_TEST_ID}>
+        <Bold>Bounty payout:</Bold>
+        <div className={classes.txDataParagraph}>
+          {data.bountyLink && (
+            <p>
+              Bounty:{' '}
+              <a href={data.bountyLink} rel="noopener noreferrer" target="_blank">
```

Comment:
Please truncate the trailing zeroes of the Byte32 correctly :)

Reply:
there should be no trailing zeros already for new payouts. Once again, it applies only to new payouts — old ones created before the fix will still have zeroes and that is expected

Reply:
I am still experiencing the same problem with a now newly setup safe.

Reply:
I would leave this open, everything else works great and if we decide to use the Safe we could investigate this further on.